config/initializers/session_store.rb

```diff
@@ -1,3 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :active_record_store, key: '_gibct-data-service_session'
+# Rails.application.config.session_store :active_record_store, key: '_gibct-data-service_session'
```

Comment:
You can probably just remove the commented out code. Git can be utilized for historical tracking purposes

Reply:
Removed, missed removal of code after testing removal of gem